### PR TITLE
Sonus restart

### PR DESCRIPTION
**Pull request by sdetweil**
####  Description
fix tab indention

####  Checklist
- [ x] I have read and understand the CONTRIBUTIONS.md file
- [ x] I have searched for and linked related issues
- [ x] I have added config.schema.json file if config option are required.
- [ x] I am NOT targeting master branch

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -62,101 +64,116 @@ function createWindow() {
...
+			mainWindow.webContents.send("partial-results", message.substring(4))
+		} else if (message.startsWith("!f:")) {
+			mainWindow.webContents.send("final-results", message.substring(4))
 		} else {
 			console.error(message.substring(3))
 		}
 	})
+    
+    // if we receive a closed event from the keyword spotter
+	kwsProcess.on("close", function() {
```
Comment (evancohen) [inline main.js]: We shouldn't need to restart Sonus anymore, it should handle things on its own:
https://github.com/evancohen/sonus/releases/tag/1.0.1

File: main.js [hunk 2]
```diff
@@ -1,36 +1,38 @@
 /* global __dirname */
 /* global process */
-const electron = require('electron')
+const electron = require("electron")
```
Comment (evancohen) [inline main.js]: Why change all the quotes from `'` to `"`?

File: main.js (hunk 1 above)
Comment (sdetweil) [reply main.js]: no.. this is another bug,.. suddenly the sonus process just disappears..  not the arecord falling out from under it..

an example.. kill the node sonus task...   how do we recover?? we don't

File: main.js (hunk 2 above)
Comment (sdetweil) [reply main.js]: i didn't.. i JUST forked and cloned... starting from scratch and edited the few lines, and pushed it back...

Comment (sdetweil) [reply main.js]: oh, i ran eslint to fix the tabs..  I bet it changed the quotes...

File: main.js [hunk 3]
```diff
@@ -1,36 +1,38 @@
...
+powerSaveBlocker.start("prevent-display-sleep")
 
 // Launching the mirror in dev mode
 const DevelopmentMode = process.argv.includes("dev")
 
 // Load the smart mirror config
 let config
 let firstRun = false
+let kwsProcess = null
+let quiting = false
```
Comment (evancohen) [inline main.js]: *quitting

Comment (evancohen) [reply main.js]: Other than that this looks good! Thanks dude!

**Pull request merged**